### PR TITLE
Update to 1.9.5

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- Update to 1.9.5

https://github.com/Toblerity/Fiona/tree/1.9.5

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -64,7 +74,7 @@ about:
     information systems with other computer systems.
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:74]: Could you add this license https://github.com/Toblerity/Fiona/blob/master/fiona/_vendor/munch/LICENSE.txt as `munch` is vendored starting from v1.9.4, see https://github.com/Toblerity/Fiona/blob/195579d8737afdd99dfab6db8315815fa12d0f18/CHANGES.txt#L42

**Review by skupr-anaconda — APPROVED** [review]
LGTM but the `munch` license

File: recipe/meta.yaml [hunk 2]
```diff
@@ -35,15 +41,19 @@ requirements:
...
+    - six
     - setuptools
     - gdal
     - {{ pin_compatible('numpy') }}
     - shapely  # only option calc we include by default
+  run_constrained:
+    - boto3 >=1.3.1
 
 test:
   source_files:
```
Comment (boldorider4) [inline recipe/meta.yaml:54]: the `commands` section is missing

Comment (skupr-anaconda) [reply recipe/meta.yaml:54]: They are placed inside `run_test.{bat|sh|py}`

File: recipe/meta.yaml (hunk 1 above)
Comment (JeanChristopheMorinPerso) [reply recipe/meta.yaml:74]: Umm, good catch.

Comment (cbouss) [reply recipe/meta.yaml:83]: thank you, addressed.

**Review by JeanChristopheMorinPerso — APPROVED** [review]

**Pull request merged**